ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/scribble/v1/scribble/scribble.html

```diff
@@ -0,0 +1,93 @@
+     data-sly-use.renderer="${'scribble.js'}"
+     data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+     data-sly-use.label="${renderer.labelPath}"
+     data-sly-use.shortDescription="${renderer.shortDescriptionPath}"
+     data-sly-use.longDescription="${renderer.longDescriptionPath}"
+     data-sly-use.errorMessage="${renderer.errorMessagePath}"
+     data-sly-use.questionMark="${renderer.questionMarkPath}"></sly>
+
+<div data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
+     class="cmp-adaptiveform-scribble"
+     data-cmp-is="adaptiveFormScribble"
+     data-cmp-visible="${scribble.visible ? 'true' : 'false'}"
+     data-cmp-enabled="${scribble.enabled ? 'true' : 'false'}"
+     data-cmp-required="${scribble.required ? 'true': 'false'}"
+     data-cmp-readonly="${scribble.readOnly ? 'true' : 'false'}"
+     id="${scribble.id}"
+     data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}"
+     data-sly-test.widgetId="${'{0}-{1}' @ format=[scribble.id, 'widget']}">
+
+    <div class="cmp-adaptiveform-scribble__label-container">
+      <div data-sly-call="${label.label @componentId=widgetId, labelValue=scribble.label.value, labelVisible=scribble.label.visible, labelRichText=scribble.label.richText, bemBlock='cmp-adaptiveform-scribble'}" data-sly-unwrap></div>
+      <div data-sly-call="${questionMark.questionMark @componentId=scribble.id, longDescription=scribble.description, bemBlock='cmp-adaptiveform-scribble'}" data-sly-unwrap></div>
+    </div>
+
+    <button type="button" class="cmp-adaptiveform-scribble__canvas-signed-container">
+      <img class="cmp-adaptiveform-scribble__canvas-signed-image" src="${scribble.value}" />
+    </button>
+
+    <div data-sly-call="${shortDescription.shortDescription @componentId=scribble.id, shortDescriptionVisible=scribble.tooltipVisible, shortDescription=scribble.tooltip, bemBlock='cmp-adaptiveform-scribble'}" data-sly-unwrap>
+    </div>
+
+    <div data-sly-call="${longDescription.longDescription @componentId= scribble.id, longDescription=scribble.description, bemBlock='cmp-adaptiveform-scribble'}" data-sly-unwrap></div>
+
+    <div data-sly-call="${errorMessage.errorMessage @componentId=scribble.id, bemBlock='cmp-adaptiveform-scribble'}" data-sly-unwrap></div>
+      <div class="cmp-adaptiveform-scribble__container" tabindex="0" role="dialog" aria-label="pleaseSignText">
+      <div class="cmp-adaptiveform-scribble__header">Please sign here</div>
+        <div class="cmp-adaptiveform-scribble__content">
+          <div class="cmp-adaptiveform-scribble__canvases">
+            <div class="cmp-adaptiveform-scribble__signcanvases" style="display:inline-block; vertical-align:top;">
+              <canvas class="cmp-adaptiveform-scribble__canvas"></canvas>
+              <input type="text" class="cmp-adaptiveform-scribble__keyboard-sign-box" name="signatureText" placeholder="Type Your Signature Here">
```

Comment:
why are we removing placeholder from dialog, when we can use it here. Also the one you have hardcode is not localized